src/Core/src/Platform/Android/SemanticExtensions.cs

```diff
@@ -100,5 +99,13 @@ public static void UpdateSemantics(this View nativeView, IView view)
 
 			ViewCompat.SetAccessibilityHeading(nativeView, semantics.IsHeading);
 		}
+
+		internal static View GetSemanticNativeElement(this View nativeView)
+		{
+			if (nativeView is AndroidX.AppCompat.Widget.SearchView sv)
+				return sv.FindViewById(Resource.Id.search_button)!;
```

Comment:
Depending on how this is used, if you have `ViewHelper.SetContentDescription()`, maybe we could also do this check and `FindByViewId()` call in Java?